doc/plugin/multigraphing.rst

```diff
@@ -9,7 +9,9 @@ As of 1.4.0 Munin supports multigraph plugins.
 What are they?
 ==============
 
-A multigraph plugin supports a "hierarchy of graphs" to provide drill-down from general graphs to more specific graphs. One of the most obvious cases for this is network switch graphing where showing per-port traffic for 48 ports in the main host view would be overwhelming. Therefore the snmp__if_multi plugin presents two graphs on the main host view: if_bytes and if_errors. If you click on the if_bytes graph you will arrive at another page showing the throughput on all interfaces. If you click on the if_errors graph you will arrive on a page showing errors on all interfaces.
+A multigraph plugin is a plugin that emits multiple graphs when fetched, each with their own labels and data. This can be used both to emit heterogenous graphs for multiple unrelated metrics, and to create graph heirarchies that permit grouping related metrics (e.g. network traffic per interface) into a single top-level graph that can then be expanded to show a collection of more detailed individual graphs.
```

Comment:
`heirarchies` -> `hierarchies`
and in the following line:
`heirarchy` -> `hierarchy`

"codespell" is your friend :)